src/gl/context.cpp

```diff
@@ -127,7 +127,7 @@ Context::Context(unsigned size)
   };
 
   auto scroll_callback = [](GLFWwindow *w, double /*xOffset*/, double yOffset) {
-    glfwWPtr(w)->viewScale += 0.1f * yOffset;
+    glfwWPtr(w)->viewScale += 0.1f * static_cast<float>(yOffset);
```

Comment:
Prefer static_cast<float>( 0.1 * yOffset)

Do the calc as double, then cast the result.

(Minor impact if any, but still ... . )

Reply:
Out of curiosity, is doing the calculation and then casting faster than the opposite way?